packages/editor/src/components/editor-canvas/index.js

```diff
@@ -308,6 +308,7 @@ function EditorCanvas( {
 				style: {
 					...iframeProps?.style,
 					...deviceStyles,
+					border: showEditorPadding ? undefined : 0,
```

Comment:
@glendaviesnz 

I was wondering whether (and how) to harmonize this with the site editor, which places the padding on a higher element using `.is-focus-mode`

https://github.com/WordPress/gutenberg/blob/b718580794ceb0a870def8119b6ef129f76486c8/packages/edit-site/src/components/block-editor/style.scss#L38-L38

I'm just not sure if moving this logic to packages/edit-post/src/components/visual-editor/index.js would affect anything.

Reply:
Asking because of the addition of `showEditorPadding` in https://github.com/WordPress/gutenberg/pull/57700

Reply:
It's tricky though because the site editor has the resize controls. But arguably they should exist in the post editor too.... there's a bit to untangle 😅

Reply:
Just merge this one :trollface:

Reply:
I think the current aim is to get the post-editor and site-editor to share as much as possible via `packages/editor` so if think get the site editor to work via `packages/editor/src/components/editor-canvas/index.js` is a better approach than moving this to `packages/edit-post/src/components/visual-editor/index.js`

Reply:
Thanks folks, I'll take a deeper look to see if it can whipped together.